test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs

```diff
@@ -7,48 +7,55 @@ namespace Microsoft.Framework.Runtime.Tests
+        [InlineData("win81", "aspnetcore50", false)]
+        [InlineData("aspnetcore50", "netcore45", false)]
+        [InlineData("aspnetcore50", "win8", false)]
+        [InlineData("aspnetcore50", "win81", false)]
+        [InlineData("aspnetcore50", "portable-net40+win8+aspnetcore50", true)]
+        // Temporary until our dependencies update
+        [InlineData("aspnetcore50", "portable-net45+win8", true)]
+        [InlineData("aspnetcore50", "portable-net451+win81", true)]
+        [InlineData("aspnetcore50", "portable-net40+sl5+win8", false)]
+        [InlineData("aspnetcore50", "portable-net45+win8", true)]
+        [InlineData("aspnetcore50", "portable-net451+win81", true)]
+        [InlineData("aspnetcore50", "portable-net40+sl5+win8", false)]
+
+        // Tests for aspnet -> dnx rename
+        [InlineData("dnx451", "aspnet50", true)]
         [InlineData("dnx451", "net45", true)]
-        [InlineData("dnxcore50", "k10", false)]
-        [InlineData("k10", "dnxcore50", false)]
-        [InlineData("netcore45", "dnxcore50", false)]
-        [InlineData("win8", "dnxcore50", false)]
-        [InlineData("win81", "dnxcore50", false)]
-        [InlineData("dnxcore50", "netcore45", false)]
-        [InlineData("dnxcore50", "win8", false)]
-        [InlineData("dnxcore50", "win81", false)]
-        [InlineData("dnxcore50", "portable-net40+win8+dnxcore50", true)]
-        [InlineData("net45", "dnx452", false)]
-        [InlineData("dnx452", "net45", true)]
-        [InlineData("dnx452", "net45", true)]
-        [InlineData("dnxcore50", "k10", false)]
-        [InlineData("k10", "dnxcore50", false)]
-        [InlineData("netcore45", "dnxcore50", false)]
-        [InlineData("win8", "dnxcore50", false)]
-        [InlineData("win81", "dnxcore50", false)]
-        [InlineData("dnxcore50", "netcore45", false)]
-        [InlineData("dnxcore50", "win8", false)]
-        [InlineData("dnxcore50", "win81", false)]
+        [InlineData("aspnet50", "dnx451", false)]
+        [InlineData("net45", "dnx451", false)]
+
+        [InlineData("dnxcore50", "aspnetcore50", true)]
+        [InlineData("aspnetcore50", "dnxcore50", false)]
+        
+        // Portable stuff?
         [InlineData("dnxcore50", "portable-net40+win8+dnxcore50", true)]
-        // Temporary until our dependencies update
+        [InlineData("dnxcore50", "portable-net40+win8+aspnetcore50", true)]
         [InlineData("dnxcore50", "portable-net45+win8", true)]
         [InlineData("dnxcore50", "portable-net451+win81", true)]
         [InlineData("dnxcore50", "portable-net40+sl5+win8", false)]
         [InlineData("dnxcore50", "portable-net45+win8", true)]
         [InlineData("dnxcore50", "portable-net451+win81", true)]
         [InlineData("dnxcore50", "portable-net40+sl5+win8", false)]
-
-        // Tests for aspnet -> dnx rename
-        // DNX packages installed into DNX451 projects? OK!
-        [InlineData("dnx451", "dnx452", true)]
```

Comment:
As discussed in JabbR, this was a bad test. They should be swapped. I'll add that in to the final merge.